Formula/xauth.rb

```diff
@@ -0,0 +1,42 @@
+class Xauth < Formula
+  desc "X.Org Applications: xauth"
+  homepage "https://www.x.org/"
+  url "https://www.x.org/pub/individual/app/xauth-1.1.tar.bz2"
+  sha256 "6d1dd1b79dd185107c5b0fdd22d1d791ad749ad6e288d0cdf80964c4ffa7530c"
+  license "MIT-open-group"
+  revision 1
```

Comment:
Why start at 1?

Reply:
This is because in the original repo, it was already at revision 1.  If we don't match the revision number already being used by our users, we'd break existing installations of this formula.

Reply:
We already have this problem for about a hundred other formulae, so we don't really need to sort it out locally for another one, I think. We'll just have to fix global behaviour in the migration (e.g. behave as if migration is like incrementing `version_scheme`).

Reply:
I got rid of the revision here.  I think is ready to go now.